daemon/apparmor_default.go

```diff
@@ -8,6 +8,8 @@ import (
 
 	"github.com/containerd/containerd/pkg/apparmor"
 	aaprofile "github.com/docker/docker/profiles/apparmor"
+	"github.com/pkg/errors"
```

Comment:
`pkg/errors` is deprecated -- we shouldn't use it in any new code. Let's use the stdlib instead: https://pkg.go.dev/errors#Is

Reply:
Thanks, good catch, will fix. My `gopls` always insists on the `pkg/errors` by default instead of the stdlib one and I missed this 🙄